components/pkg-export-kubernetes/tests/KubernetesManifestTest.yaml

```diff
@@ -0,0 +1,38 @@
+---
+## Secret for initial configuration.
+#apiVersion: v1
+#kind: Secret
+#metadata:
+#  name: user-toml-secret
+#type: Opaque
+#data:
+#  # Each configuration item needs to be encoded in base64.
+#  # Plain text content of the secret: "port = 4444"
+#  user.toml: cG9ydCA9IDQ0NDQ=
+#---
+apiVersion: habitat.sh/v1
+kind: Habitat
+metadata:
+  ## Name of the Habitat service.
```

Comment:
Why use two `#`?

Reply:
I just checked from git blame, it blames @krnowak for this. :) i-e Two # were used in the first manifest template string used in 8598b4137d0ec14def123acb5f77ba9ee8b39297.

Reply:
OK, should be removed then I guess?